docs/components/connectors/introduction.md

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
Is this a new document, or has it been rewritten? I'm trying to understand if doc has move as part of the top level changes or whether I should comment on the content here

Reply:
It has been rewritten/expanded to focus solely on Connectors (this was the Integration Framework introduction previously). I think we can treat it as a new document and review it firmly. It is THE high-level entry point for the Connectors as a whole.

Reply:
Ok thanks - in that case I need some time to review the top level description. 

Generally agree with the approach of top-level 'connectors' rather than integration framework, although we also have marketing blurb that refers to the integration framework which would be inconsistent (see Bernd here: https://camunda.com/platform/modeler/connectors/).

Before we commit, lets have a quick conversation with @amyjohnston in marketing

Reply:
Yes, I agree. I discussed this with Preeti, this are the changes I proposed for that page:

1-----------------------------
Connect to all the things.
Use out-of-the-box connectors or build your own using Camunda’s Integration Framework to connect to everything, including:
==>
Connect to all the things.
Use out-of-the-box connectors or build your own using Camunda’s Connector SDK to connect to everything, including:

2-----------------------------
Integration Framework
Build your own connectors.
==>
Connector SDK
Build your own connectors.

3-----------------------------
The Integration Framework consists of two components, [Connector Templates](https://docs.camunda.io/docs/components/integration-framework/connectors/custom-built-connectors/connector-templates/) and the [Connector SDK](https://docs.camunda.io/docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk/).
==> Connectors consist of two components, [Connector Templates](https://docs.camunda.io/docs/components/integration-framework/connectors/custom-built-connectors/connector-templates/) and the [Connector Function](https://docs.camunda.io/docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk/#runtime-logic).

4-----------------------------
The Connector SDK provides a simple, agnostic approach to implementing custom Connectors across both SaaS, self-managed, and hybrid environments.
==> The [Connector SDK](https://docs.camunda.io/docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk/) provides a simple, agnostic approach to implementing custom Connector Functions that run in SaaS, self-managed, and hybrid environments.